packages/rolldown/src/binding.d.ts

```diff
@@ -626,7 +652,7 @@ export interface IsolatedDeclarationsOptions {
 export interface IsolatedDeclarationsResult {
   code: string
   map?: SourceMap
-  errors: Array<string>
+  errors: Array<Error>
```

Comment:
Here is wrong I guess. https://github.com/rolldown/rolldown/pull/3104#issuecomment-2532213868